### PR TITLE
Adding Ola Sendecka to Team members

### PR DESCRIPTION
**Pull request by asendecka**
Better late than never!

### CODE REVIEW

File: docs/internals/team.txt [hunk 1]
```diff
@@ -555,6 +555,32 @@ Daniele Procida
     .. _Django Girls: https://djangogirls.org
     .. _Potato: https://p.ota.to
 
+`Ola Sendecka`_
+    Ola started her adventure with Django when writing university project in
```
Comment (bmispelon) [inline docs/internals/team.txt]: writing **a** university project

File: docs/internals/team.txt [hunk 2]
```diff
@@ -555,6 +555,32 @@ Daniele Procida
...
+    2009. After that she attended her first DjangoCon Europe and that
+    defined her future as a Django professional.
+
+    Since then she is an active community member. Ola co-organized a number of
```
Comment (bmispelon) [inline docs/internals/team.txt]: she **has been** an active community member

File: docs/internals/team.txt [hunk 3]
```diff
@@ -555,6 +555,32 @@ Daniele Procida
...
+    Django sprints and conferences including:
+    `DjangoCon Europe 2013`_, `Django Under the Hood 2015`_ and
+    `DjangoCon Europe 2016`_. Together with Ola Sitarska she founded
+    `Django Girls`_, co-authored the `Django Girls Tutorial`_ and
+    is a member of Django Girls Foundation. She is also an author of
```
Comment (bmispelon) [inline docs/internals/team.txt]: member of **the** Django Girls Foundation

File: docs/internals/team.txt [hunk 4]
```diff
@@ -555,6 +555,32 @@ Daniele Procida
...
+    `Coding is for Girls`_ YouTube series teaching programming and Django
```
Comment (bmispelon) [inline docs/internals/team.txt]: author of **the** Coding is for Girls YouTube serie

File: docs/internals/team.txt [hunk 5]
```diff
@@ -555,6 +555,32 @@ Daniele Procida
     .. _Django Girls: https://djangogirls.org
     .. _Potato: https://p.ota.to
 
+`Ola Sendecka`_
+    Ola started her adventure with Django when writing university project in
+    2009. After that she attended her first DjangoCon Europe and that
```
Comment (MarkusH) [inline docs/internals/team.txt]: maybe "... Europe which defined ..." or "... 2009. Her attendance at the next DjangoCon Europe defined ..."

File: docs/internals/team.txt [hunk 6]
```diff
@@ -555,6 +555,32 @@ Daniele Procida
...
+    defined her future as a Django professional.
```
Comment (MarkusH) [inline docs/internals/team.txt]: "... future live as a ..." ?

**Pull request merged**